llumnix/backends/vllm/worker.py

```diff
@@ -66,135 +84,63 @@ def init_migration(self, num_migration_cache_blocks: int, src_worker_handle_list
                 node_id=ray.get_runtime_context().get_node_id(),
                 soft=False,
             )
-        self.recv_actor = RecvActor.options(scheduling_strategy=scheduling_strategy).remote()
 
-        self.migration_stream = torch.cuda.Stream()
-        self.default_stream = torch.cuda.current_stream()
-        self.num_migration_cache_blocks = num_migration_cache_blocks
-        assert self.migration_stream != self.default_stream
         pin_memory = is_pin_memory_available()
         if not pin_memory:
             # Pinning memory in WSL is not supported.
             # https://docs.nvidia.com/cuda/wsl-user-guide/index.html#known-limitations-for-linux-cuda-applications
             logger.warning("Using 'pin_memory=False' as WSL is detected. "
-                           "This may slow down the performance.")
-        migration_cache_size = self.cache_engine.block_size * self.cache_engine.num_heads * self.cache_engine.head_size
+                            "This may slow down the performance.")
+
+        num_migration_cache_blocks = migration_config.migration_cache_blocks
```

Comment:
num_migration_cache_blocks is never used below except for assigment, why not directly assign self.num_migration_cache_blocks?